modules/ext.thanks.corethank.js

```diff
@@ -134,4 +140,18 @@
 	mw.hook( 'wikipage.content' ).add( ( $content ) => {
 		addActionToLinks( $content );
 	} );
+
+	mw.hook( 'wikipage.content' ).add( function ( $content ) {
+		addActionToLinks( $content );
+		reloadThankedState();
+	} );
```

Comment:
Bad cherry pick? This is called twice (1st time in L140). I think the change should include the call to `reloadThankedState() in the first call without re-declaring this hook